apps/prairielearn/src/server.js

```diff
@@ -393,8 +393,14 @@ module.exports.initExpress = function () {
     workspaceProxy,
   ]);
 
-  // Limit to 5MB of JSON
-  app.use(bodyParser.json({ limit: 5 * 1024 * 1024 }));
+  app.use((req, res, next) => {
+    // Stripe webhook signature verification requires the raw body, so we avoid
+    // using the body parser for that route.
+    if (req.path === '/pl/webhooks/stripe') return next();
```

Comment:
I wish there was a better way to handle this!

Reply:
I was hoping that the body parser might retain a copy of the raw body somewhere that we could use, but no luck!